.travis/linux.install.sh

```diff
@@ -30,12 +30,12 @@ sudo apt-get install --no-install-recommends \
     libxmu-dev \
     python \
     python-wxgtk3.0 \
+    python-six \
     python-dev \
     unixodbc-dev \
     libnetcdf-dev   \
     netcdf-bin \
     dpatch \
     libblas-dev \
     liblapack-dev \
-    python-numpy
-
+    python-numpy
```

Comment:
Whitespace change introducing no new line at the end of file, fix:

```suggestion
    python-numpy
    
```